### PR TITLE
Slim navigation

### PR DESCRIPTION
**Pull request by apepper**
Next iteration of the previous "portal-only" version.

### CODE REVIEW

File: design/images/logo-jr-dark.svg
Comment (dcsaszar) [inline design/images/logo-jr-dark.svg:1]: * [ ] FYI weird to see JustRelate logos on Tynacoon

File: src/Widgets/NavigationWidget/NavigationWidgetEditingConfig.ts [hunk 1]
```diff
@@ -3,9 +3,20 @@ import { NavigationWidget } from './NavigationWidgetClass'
 import Thumbnail from './thumbnail.svg'
 
 provideEditingConfig(NavigationWidget, {
-  attributes: { logOutLabel: { title: 'Log-out menu item label' } },
+  attributes: {
+    logOutLabel: { title: 'Log-out menu item label' },
+    slimDesign: {
+      title: 'Slim design?',
+      description: 'Shows only the meta navigation, not the "main" navigation.',
```
Comment (dcsaszar) [inline src/Widgets/NavigationWidget/NavigationWidgetEditingConfig.ts:10]: * [ ] Why is main in quotes? Maybe use _hierarchy_ to make clearer what main refers to?

File: src/Widgets/NavigationWidget/SubComponents/MainNavigation.tsx [hunk 2]
```diff
@@ -3,14 +3,19 @@ import Nav from 'react-bootstrap/Nav'
...
 export const MainNavigation = connect(function MainNavigation({
   root,
   searchInputLabel,
+  widget,
 }: {
   root: HomepageInstance
   searchInputLabel: string
+  widget: NavigationWidgetInstance
 }) {
+  if (widget.get('slimDesign')) return null
```
Comment (dcsaszar) [inline src/Widgets/NavigationWidget/SubComponents/MainNavigation.tsx:17]: * [ ] Adding a widget prop in order to access a condition the container could check makes not much sense. Move the condition up?

File: src/Widgets/NavigationWidget/SubComponents/MetaNavigation.tsx [hunk 3]
```diff
@@ -57,10 +57,13 @@ export const MetaNavigation = connect(function MetaNavigation({
 
 const PortalLink = connect(function PortalLink({
   root,
+  widget,
 }: {
   root: HomepageInstance
   widget: NavigationWidgetInstance
 }) {
+  if (widget.get('slimDesign')) return null
```
Comment (dcsaszar) [inline src/Widgets/NavigationWidget/SubComponents/MetaNavigation.tsx:65]: * [ ] See other comment regarding container vs child logic and added `widget` prop.

File: src/Widgets/NavigationWidget/NavigationWidgetEditingConfig.ts [hunk 4]
```diff
@@ -3,9 +3,20 @@ import { NavigationWidget } from './NavigationWidgetClass'
...
+    },
+  },
   title: 'Navigation',
   thumbnail: Thumbnail,
-  properties: ['metaNavigationObjs', 'searchInputLabel', 'logOutLabel'],
+  properties: [
+    'metaNavigationObjs',
+    'slimDesign',
+    'searchInputLabel',
```
Comment (dcsaszar) [inline src/Widgets/NavigationWidget/NavigationWidgetEditingConfig.ts:18]: * [ ] The search is currently hidden if we use the slim nav. If this is intentional, let's please grey out the label property conditionally.

**Review by dcsaszar — CHANGES_REQUESTED** [review]
Nice, see comments.

File: src/Widgets/NavigationWidget/NavigationWidgetEditingConfig.ts (hunk 4 above)
Comment (apepper) [reply src/Widgets/NavigationWidget/NavigationWidgetEditingConfig.ts:18]: => https://github.com/Scrivito/scrivito-portal-app/pull/480

**Review by dcsaszar — APPROVED** [review]

**Pull request merged**